.circleci/regenerate.py

```diff
@@ -45,13 +45,19 @@ def build_workflows(prefix='', filter_branch=None, upload=False, indentation=6,
                             (python_version != python_versions[-1] or
                              (cu_version not in [cu_versions[0], cu_versions[-1]])):
                             fb = "master"
+                        if not fb and (os_type == 'linux' and
+                                       cu_version == 'cpu' and
+                                       btype == 'wheel' and
+                                       python_version == '3.7'):
+                            # the fields must match the build_docs "requires" dependency
+                            fb = "*"
                         w += workflow_pair(
                             btype, os_type, python_version, cu_version,
                             unicode, prefix, upload, filter_branch=fb)
 
     if not filter_branch:
         # Build on every pull request, but upload only on nightly and tags
-        w += build_doc_job(None)
+        w += build_doc_job('nightly')
```

Comment:
@mattip does this mean that the docs are not build on every PR anymore?

Reply:
hmm. It seems to be building on PRs, for instance taking a random PR and looking at the [CI build](https://app.circleci.com/pipelines/github/pytorch/vision/8948/workflows/e9471ac9-00fd-4c07-993b-dbff08add903) I see a `build_docs` run 2/3 of the way down. Although I agree the filter says only nightly, so it is misleading.

```
      - build_docs:
          filters:
            branches:
              only:
              - nightly
            tags:
              only: /v[0-9]+(\.[0-9]+)*-rc[0-9]+/
          name: build_docs
          python_version: '3.7'
          requires:
          - binary_linux_wheel_py3.7_cpu

```

Reply:
I'm afraid the PR you're linking to was pushed before this one got merged.

In more recent commits, e.g. https://app.circleci.com/pipelines/github/pytorch/vision?branch=pull%2F3808 from this morning, I do not see the `build_docs` job.